testsuite/elytron/src/test/resources/logging.properties

```diff
@@ -0,0 +1,31 @@
+# Additional loggers to configure (the root logger is always configured)
```

Comment:
It allows to control what's included in the log on the client side. E.g. when debugging SASL mechanisms, I want to see sth like:

```
2017-06-19 16:47:45,584 TRACE [org.wildfly.security] (main) getAuthenticationConfiguration uri=remote://127.0.0.1:10567, protocolDefaultPort=-1, abstractType=null, abstractTypeAuthority=null, MatchRule=[], AuthenticationConfiguration=[AuthenticationConfiguration:principal=digest-md5,set-host=127.0.0.1,set-port=10567,credentials-present,providers-supplier=org.wildfly.security.util.ProviderUtil$1@53f0a4cb,sasl-mechanism-selector=DIGEST-MD5,mechanism-properties={wildfly.sasl.local-user.quiet-auth=true}]
2017-06-19 16:47:45,592 TRACE [org.wildfly.security] (main) getAuthenticationConfiguration uri=remote://127.0.0.1:10567, protocolDefaultPort=-1, abstractType=null, abstractTypeAuthority=null, MatchRule=[], AuthenticationConfiguration=[AuthenticationConfiguration:principal=digest-sha,set-host=127.0.0.1,set-port=10567,credentials-present,providers-supplier=org.wildfly.security.util.ProviderUtil$1@8c11eee,sasl-mechanism-selector=DIGEST-SHA,mechanism-properties={wildfly.sasl.local-user.quiet-auth=true}]
2017-06-19 16:47:45,592 TRACE [org.wildfly.security] (main) getAuthenticationConfiguration uri=remote://127.0.0.1:10567, protocolDefaultPort=-1, abstractType=null, abstractTypeAuthority=null, MatchRule=[], AuthenticationConfiguration=[AuthenticationConfiguration:principal=digest-sha,set-host=127.0.0.1,set-port=10567,credentials-present,providers-supplier=org.wildfly.security.util.ProviderUtil$1@8c11eee,sasl-mechanism-selector=DIGEST-SHA,mechanism-properties={wildfly.sasl.local-user.quiet-auth=true}]
2017-06-19 16:47:45,599 TRACE [org.wildfly.security] (main) getAuthenticationConfiguration uri=remote://127.0.0.1:10567, protocolDefaultPort=-1, abstractType=null, abstractTypeAuthority=null, MatchRule=[], AuthenticationConfiguration=[AuthenticationConfiguration:principal=digest-sha-256,set-host=127.0.0.1,set-port=10567,credentials-present,providers-supplier=org.wildfly.security.util.ProviderUtil$1@333dd51e,sasl-mechanism-selector=DIGEST-SHA-256,mechanism-properties={wildfly.sasl.local-user.quiet-auth=true}]
```

So I just need to change level from `INFO` to `TRACE` in the `logging.properties`.